WiserTaskScheduler/WiserTaskScheduler/Modules/DocumentStoreRead/Models/DocumentStoreReadModel.cs

```diff
@@ -23,4 +23,10 @@ public class DocumentStoreReadModel : ActionModel
     /// Gets or sets the user name
     /// </summary>
     public string UsernameForLogging { get; set; } = "WTS (Document Store Read)";
+    
+    /// <summary>
+    /// Set or sets what published Environment the item needs to be set as
+    /// </summary>
+    public int? SetPublishedEnvironment { get; set; } = null;
```

Comment:
Please change the type directly to the enum. In the XML people can still use the integers to set multiple flags.

Please rename the property to "PublishedEnvironmentToSet", currently it sounds like a function name.